src/handlers/generate-erc20-permit.ts

```diff
@@ -54,6 +54,8 @@ export async function generateErc20PermitSignature(
       issueNodeId = contextOrPayload.payload.issue.node_id;
     } else if ("pull_request" in contextOrPayload.payload) {
       issueNodeId = contextOrPayload.payload.pull_request.node_id;
+    } else if (contextOrPayload.config.runId) {
+      issueNodeId = contextOrPayload.config.runId.toString();
```

Comment:
The node ID should not be the CI run ID? This doesnt make any sense to me

Reply:
Because when you are manually generating a permit with Action workflow dispatch, it's not associated to any issue so you have to use a unique nonce - that could be the run ID or just a uuid

Reply:
Other ideas could be:

- Pulling it from a workflow input, like `uuid`
- Setting a fixed value that always gives the same nonce each time